opencog/attention/MinMaxSTIUpdatingAgent.h

```diff
@@ -0,0 +1,102 @@
+/*
+ * opencog/attention/MinMaxSTIUpdatingAgent.h
+ *
+ * Copyright (C) 2008 by OpenCog Foundation
+ * Written by Joel Pitt <joel@fruitionnz.com>
```

Comment:
This seems to be a new file .. did Joel Pitt actually write it 8 years ago? I understand you might be doing a lot of cut-n-paste editing but once the changes are fairly lasrge, preserving the cut-n-paste author seems incorrect.  Its not just this file, it seems to be several of them.